machine/quanta/quanta_bwde/rootconf/sysroot-lib-onie/sysinfo-platform

```diff
@@ -0,0 +1,21 @@
+
+#  Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+get_serial_num()
+{
+    [ -x /usr/bin/onie-syseeprom ] && /usr/bin/onie-syseeprom -g 0x23
+}
```

Comment:
This function is not needed, as the grub-arch default implementation does the same thing:
https://github.com/opencomputeproject/onie/blob/master/rootconf/grub-arch/sysroot-lib-onie/sysinfo-arch#L11